middleware/internal/greeter/greeter.pb.go

```diff
@@ -6,9 +6,10 @@ package greeter
 import (
 	context "context"
 	fmt "fmt"
+	math "math"
```

Comment:
Please revert this change since this is generated code.

Reply:
DOne